docs/shared/angular-standalone-tutorial/angular-standalone.md

```diff
@@ -0,0 +1,795 @@
+              },
+              {
+                "sourceTag": "scope:orders",
+                "onlyDependOnLibsWithTags": [
+                  "scope:orders",
+                  "scope:products",
+                  "scope:shared"
+                ]
+              },
+              {
+                "sourceTag": "scope:products",
+                "onlyDependOnLibsWithTags": ["scope:products", "scope:shared"]
+              },
+              {
+                "sourceTag": "scope:shared",
+                "onlyDependOnLibsWithTags": ["scope:shared"]
+              }
+            ]
+          }
+        ]
+      }
+    },
+    ...
+  ]
+}
+```
+
+To test it, go to your `modules/products/src/lib/products/products.component.ts` file and import the `OrderComponent` from the `modules-orders` project:
+
+```tsx {% fileName="modules/products/src/lib/products/products.component.ts" %}
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+// 👇 this import is not allowed
+import { OrdersComponent } from '@myngapp/modules/orders';
+
+@Component({
+  selector: 'myngapp-products',
+  standalone: true,
+  imports: [CommonModule, OrdersComponent],
+  templateUrl: './products.component.html',
+  styleUrls: ['./products.component.css'],
+})
+export class ProductsComponent {}
+```
+
+If you lint your workspace you'll get an error now:
+
+```shell {% command="nx run-many -t lint" %}
+    ✖  nx run modules-products:lint
+       Linting "modules-products"...
+
+       /Users/juri/nrwl/content/myngapp/modules/products/src/lib/products/products.component.ts
+         3:1  error  A project tagged with "scope:products" can only depend on libs tagged with "scope:products", "scope:shared"  @nx/enforce-module-boundaries
+
+       ✖ 1 problem (1 error, 0 warnings)
+
+       Lint errors found in the listed files.
+
+    ✔  nx run modules-orders:lint (1s)
+    ✔  nx run myngapp:lint (1s)
+    ✔  nx run e2e:lint (682ms)
+    ✔  nx run modules-shared-ui:lint (797ms)
+
+ —————————————————————————————————————————————————————————————————————
+
+ >  NX   Ran target lint for 5 projects (2s)
+
+    ✔    4/5 succeeded [0 read from cache]
+
+    ✖    1/5 targets failed, including the following:
+         - nx run modules-products:lint
+
+```
+
+If you have the ESLint plugin installed in your IDE you should immediately see an error:
+
+![ESLint module boundary error](/shared/images/tutorial-angular-standalone/boundary-rule-violation-vscode.png)
+
+Learn more [about how to enforce module boundaries](/core-features/enforce-project-boundaries).
+
+## Next Steps
+
+Congrats, you made it!! You now know how to leverage the Nx standalone applications preset to build modular Angular applications.
+
+Here's some more things you can dive into next:
+
+- [Learn how to setup Tailwind](/recipes/other/using-tailwind-css-with-angular-projects)
+- [Setup Storybook for our shared UI library](/packages/storybook/documents/overview-angular)
```

Comment:
Tailwind and storybook don't seem quite as useful as these links below:

- Dig into the underlying [mental model of Nx](https://github.com/concepts/mental-model)
- Understand the [differences between the Angular CLI and Nx](https://github.com/concepts/angular-and-nx)
- Learn how to [migrate your existing Angular CLI repo to Nx](https://github.com/recipes/adopting-nx/migration-angular)

Reply:
good point, let me summarize maybe Tailwind & Storybook into a single point (to have an example of popular generators) and add some of your links